rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```
@@ -186,6 +317,48 @@ components:
       maxLength: 80
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    MetadataInformationResponse:
+      type: object
+      required:
+        - transformationStatus
+        - metadataInformations
+      properties:
+        transformationStatus:
+          $ref: '#/components/schemas/Status'
+        metadataInformations:
+          type: array
+          items:
+            $ref: '#/components/schemas/MetadataInformation'
+    MetadataInformation:
+      type: object
+      required:
+        - metadataType
+        - transformationStatus
+      properties:
+        metadataType:
```

```suggestion
        type:
```

No need to repeat metadata since this is already part of the metadata info model

---

This conflicts with the keyword "type" in these APIs. I want to avoid that.

---

What other `type`? This one is inside the `MetadataInformation` model and there is no other `type` there?

---

I'm saying that `type` is a keyword used in the JSON spec that OpenAPI uses, so the word "type" is littered through this file. I'd rather distinguish the property for readability purposes.